app/controllers/OwnBrandsController.scala

```diff
@@ -53,7 +53,10 @@ class OwnBrandsController @Inject()(
         case Some(value) => form.fill(value)
       }
 
-      Ok(view(preparedForm, mode))
+        request.userAnswers.getOrElse(UserAnswers(id = request.sdilEnrolment)).submitted match {
```

Comment:
Why not add checkSub to the controller instead?

Reply:
This one is because we don't have any userAnswers sometimes for this page so I had to do a get or else as it was optional but on the other pages it's not optional so I could use the checker